ui/app/AppLayouts/Profile/popups/BackupSeedModal.qml

```diff
@@ -136,6 +137,9 @@ StatusStackModal {
             titleText: qsTr("Confirm word #%1 of your seed phrase").arg(d.firstRandomNo + 1)
             wordRandomNumber: d.firstRandomNo
             wordAtRandomNumber: root.privacyStore.getMnemonicWordAtIndex(d.firstRandomNo)
+            onEnterPressed: {
+                nextButton.clicked();
```

Comment:
I think we need to check first if `nextButton` is enabled

Reply:
@alexjba you think `inputText.valid` is not enough? (see `BackupSeedStepBase.qml`)

Reply:
Moving the `inputText.valid` out of `BackupSeedStepBase.qml ` would make it obvious on what is the condition in which `clicked` is sent. But it's up to you. No strong opinion on it.